### PR TITLE
[receiver/mysql] Add mysql.joins metric

### PR DESCRIPTION
**Pull request by sumo-drosiek**
**Description:**

add mysql.joins metric

**Link to tracking Issue:**
https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14138

**Testing:**
unit tests

**Documentation:**
`metadata.yaml`

### CODE REVIEW

File: receiver/mysqlreceiver/metadata.yaml [hunk 1]
```diff
@@ -279,3 +283,13 @@ metrics:
       monotonic: false
       aggregation: cumulative
     attributes: [threads]
+  mysql.select:
```
Comment (djaglowski) [inline receiver/mysqlreceiver/metadata.yaml:286]: `mysql.joins`? 

`select` sounds like a count of select statements

Comment (sumo-drosiek) [reply receiver/mysqlreceiver/metadata.yaml]: agree, done

**Review by djaglowski — APPROVED** [review]

**Comment by djaglowski** [comment]
LGTM, but please resolve conflicts

**Comment by djaglowski** [comment]
Closed based on #15366

**Pull request closed without merging**